# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/angle/targets/libEGL/moz.build
# Commit: 13282d7a47a5
# Full Hash: 13282d7a47a57a73be8d060efd9645b4d642f03e
# Author: Miko Mynttinen <mikokm@gmail.com>
# Date: 2019-09-08 09:46:52
# Regressor Bug: 1578576
# File Overlap Count: 4
# Description:
#   Bug 1578576 - Part 2: Update to ANGLE 3865 r=jgilbert
#   
#   Depends on D44561
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D44579
# ==============================================================================

diff -r 3064469c073d -r 13282d7a47a5 gfx/angle/targets/libEGL/moz.build
--- a/gfx/angle/targets/libEGL/moz.build	Sat Sep 07 13:36:46 2019 +0000
+++ b/gfx/angle/targets/libEGL/moz.build	Sat Sep 07 13:38:36 2019 +0000
@@ -20,7 +20,7 @@
 DEFINES['LIBEGL_IMPLEMENTATION'] = True
 DEFINES['NOMINMAX'] = True
 #DEFINES['NO_TCMALLOC'] = True
-DEFINES['NTDDI_VERSION'] = '0x0A000003'
+DEFINES['NTDDI_VERSION'] = 'NTDDI_WIN10_RS2'
 #DEFINES['PSAPI_VERSION'] = '2'
 #DEFINES['SAFE_BROWSING_CSD'] = True
 #DEFINES['SAFE_BROWSING_DB_LOCAL'] = True
@@ -53,7 +53,6 @@
 
 #CXXFLAGS += [
 #    '/bigobj',
-#    '/d2FastFail',
 #    '/D__DATE__=',
 #    '/D__TIME__=',
 #    '/D__TIMESTAMP__=',
@@ -111,6 +110,7 @@
 #    '/we4244',
 #    '/we4456',
 #    '/we4458',
+#    '/we4715',
 #    '/we4800',
 #    '/we4838',
 #    '/WX',
@@ -165,7 +165,6 @@
 #LDFLAGS += [
 #    '/DEBUG',
 #    '/DYNAMICBASE',
-#    '/fastfail',
 #    '/FIXED:NO',
 #    '/ignore:4199',
 #    '/ignore:4221',